CHANGELOG.md

```diff
@@ -40,5 +42,6 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 - Updated README with examples for the new provider ([MODULES-6264](https://tickets.puppetlabs.com/browse/MODULES-6264))
 - Updated acceptance tests for the new provider ([MODULES-6362](https://tickets.puppetlabs.com/browse/MODULES-6362))
 
-[Unreleased]: https://github.com/puppetlabs/puppetlabs-scheduled_task/compare/0.1.0...HEAD
+[Unreleased]: https://github.com/puppetlabs/puppetlabs-scheduled_task/compare/0.2.0...HEAD
```

Comment:
Note that neither this line nor the next will produce a working link until _after_ we tag for the `0.2.0` release.